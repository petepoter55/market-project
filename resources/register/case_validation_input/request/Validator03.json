{
  "username" : "Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567",
  "password" : "Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567!",
  "email": "Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567",
  "confirmEmail" : "Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567",
  "telephone": "Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567",
  "refNoInformation" : "Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567Scb1234567"
}