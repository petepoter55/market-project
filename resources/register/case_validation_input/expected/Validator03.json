[
  "username: size must be between 1 and 100",
  "password: size must be between 1 and 100",
  "email: size must be between 1 and 50",
  "confirmEmail: size must be between 1 and 50",
  "telephone: size must be between 1 and 10",
  "refNoInformation: size must be between 1 and 10"
]